### PR TITLE
Only build travis on master branch

### PR DESCRIPTION
**Pull request by matthewrsj**
Let's not build travis every time someone pushes to their personal branch. This PR is kind of a test to make sure Travis will still run for PRs.

Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

**Comment by matthewrsj** [comment]
Also skip more pkg_integrity tests that rely on external network (can't reach pypi.debian.net right now).

File: .travis.yml [hunk 1]
```diff
@@ -6,3 +6,7 @@ python:
 script:
   - make check
   - make unittests
+
+branches:
+  only:
+    - master
```
Comment (phmccarty) [inline .travis.yml]: This change will affect autospec forks, right? I generally like to see Travis results when I'm working on a topic branch in a fork...

Comment (matthewrsj) [reply .travis.yml]: ah, that's a good point. This was just so I didn't get emails for everyone's travis builds in this repo. I'll have to rethink this :(

**Comment by matthewrsj** [comment]
Ok, I'll deal with it or come up with another solution.

**Pull request closed without merging**